expression/column.go

```diff
@@ -313,17 +314,18 @@ func (col *Column) HashCode(_ *stmtctx.StatementContext) []byte {
 }
 
 // ResolveIndices implements Expression interface.
-func (col *Column) ResolveIndices(schema *Schema) Expression {
+func (col *Column) ResolveIndices(schema *Schema) (Expression, error) {
 	newCol := col.Clone()
-	newCol.resolveIndices(schema)
-	return newCol
+	err := newCol.resolveIndices(schema)
+	return newCol, err
 }
 
-func (col *Column) resolveIndices(schema *Schema) {
+func (col *Column) resolveIndices(schema *Schema) error {
 	col.Index = schema.ColumnIndex(col)
 	if col.Index == -1 {
-		log.Errorf("Can't find column %s in schema %s", col, schema)
+		return errors.Trace(errors.Errorf("Can't find column %s in schema %s", col, schema))
```

Comment:
remove errors.Trace?

Reply:
Errorf won't add `StackTree` automatically.
And error stack is helpful for this kind of error.

Reply:
`errors.Trace()` will be removed in the near future. It's better to generate an error with stack and remove the `errors.Trace()` function call.

Reply:
`errors.ErrorfWithStack(format string, args ...interface{})`?
`pingcap/errors` have a method named `WithStack(err)` which i think it's the same thing with `errors.Trace`